export const addData=async(req,res)=>{
    try{

    }
    catch(err){
        console.log(err)
    }
}